utils/setools/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=setools
+PKG_VERSION:=4.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/setools/releases/download/4.3.0
+PKG_HASH:=315df3ae0eb29b399123c5e3330480c5d1c0da038671c9fd62a439c49a6f9105
+PKG_BUILD_DIR:=$(BUILD_DIR)/setools
+PKG_BUILD_REQUIRES:=cython
```

Comment:
I don't think this `PKG_BUILD_REQUIRES` is necessary.

Reply:
Done!